Shvachko_HM4.cpp

```diff
@@ -0,0 +1,93 @@
+#include <ctime>
+#include <iostream>
+
+using namespace std;
+
+class Array {
+private:
+  int *arr;
+  int size;
+
+public:
+  Array() //конструктор без параметров
+  {
+    size = 1;
+    arr = new int[size];
```

Comment:
```suggestion
    arr = new int[size]{};
```
вам не нужно будет писать ниже цикл